resources/sass/_projects.scss

```diff
@@ -114,6 +114,66 @@ $search-box-height: 42px;
   }
 }
 
+.loading-indicator {
+  position: absolute;
+  left: initial;
+  right: 20px;
+  bottom: 25px;
+
+  z-index: 10;
+
+  width: 170px;
+  height: 45px;
+  padding-left: 10px;
+  padding-right: 10px;
+
+  background-color: #666666;
+  border-radius: 4px;
+
+  font-weight: 500;
```

Comment:
Fair enough, can you define a `$semi-bold` constant in `_defs.scss` and use that instead then? We need to review font styles with @jkicillof at some point anyway.